sdk/storage/azure-storage-queue/azure/storage/queue/aio/_queue_client_async.py

```diff
@@ -37,7 +37,7 @@
     from .._models import QueueProperties
 
 
-class QueueClient(AsyncStorageAccountHostsMixin, QueueClientBase, StorageEncryptionMixin):
+class QueueClient(AsyncStorageAccountHostsMixin, QueueClientBase, StorageEncryptionMixin):  # type: ignore
```

Comment:
Did you try the less aggressive ignore Krista suggested? `# type: ignore[override]`. This will only ignore override type errors and type check other stuff.